tests/Endpoints/SessionTest.php

```diff
@@ -213,27 +211,13 @@ public function testDeletingSessionLearningMaterialUpdatesSessionStamp()
         $this->relatedTimeStampDeleteTest(1, 'sessionlearningmaterials', $data['id']);
     }
 
-    public function testDeletingSessionDescriptionUpdatesSessionStamp()
-    {
-        $dataLoader = $this->getContainer()->get(SessionDescriptionData::class);
-        $data = $dataLoader->getOne();
-        $this->relatedTimeStampDeleteTest($data['session'], 'sessiondescriptions', $data['id']);
-    }
-
-    public function testUpdatingSessionDescriptionUpdatesSessionStamp()
-    {
-        $dataLoader = $this->getContainer()->get(SessionDescriptionData::class);
-        $data = $dataLoader->getOne();
-        $data['description'] = 'new description';
-        $this->relatedTimeStampUpdateTest($data['session'], 'sessiondescriptions', 'sessionDescription', $data);
-    }
-
     public function testSendingNullForSessionDescription()
```

Comment:
Now that this is just an attribute I think this should be consolidated in the `putsToTest` instead of it's own section.

Reply:
consolidated. interesting side-note: sending `NULL` for description return a blank string. The caused by purifying a null value in the denormalizer here: https://github.com/ilios/ilios/blob/master/src/Denormalizer/EntityDenormalizer.php#L197

Will create a follow-up ticket.